app/javascript/packs/applications.js

```diff
@@ -1,5 +1,10 @@
-import { initialize as planSelector } from '../src/Applications/plan_selector'
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { PlanSelector } from '../src/Common/PlanSelector'
 
 document.addEventListener('DOMContentLoaded', () => {
-  window.planSelector = planSelector
+  const node = document.getElementById('plan_selector')
+  const { plans } = JSON.parse(node.getAttribute('data'))
```

Comment:
There is no attribute `data`.

Reply:
See https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes and https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*